packages/api/methods.ts

```diff
@@ -208,3 +208,11 @@ export function updateRule(rule) {
 export function deleteRule(id) {
   return send('api/rule-delete', { id });
 }
+
+export function holdForNextMonth(month, amount) {
+  return send('api/hold-for-next-month', { month, amount });
+}
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏IMO it would be _slightly_ better to be more explicit here. Otherwise: what are we holding? Transactions? Schedules? Rules? :) 

```suggestion
export function holdBudgetForNextMonth(month, amount) {
  return send('api/hold-budget-for-next-month', { month, amount });
}
```

Reply:
I think your suggestions make sense. Just wanted to keep the same naming as the other places but the other places have /budget to give some context